tests/TestCase.php

```diff
@@ -28,6 +28,8 @@ protected function defineEnvironment($app)
         $app['config']->set([
             'auth.providers.users.model' => 'Workbench\\App\\User',
             'database.default' => 'testing',
+            'cachet.path' => 'status',
+            'cache.default' => 'array',
```

Comment:
Adding this ensure path will always set to `status` and rate-limiter tests using `array` cache (reset between tests)